app/assets/javascripts/foreman_virt_who_configure/config_edit.js

```diff
@@ -1,3 +1,20 @@
+function virt_who_update_listing_mode() {
+  filtering_mode = $('#foreman_virt_who_configure_config_listing_mode').val();
+  var whitelist = $('#foreman_virt_who_configure_config_whitelist').parents('div.form-group');
+  var blacklist = $('#foreman_virt_who_configure_config_blacklist').parents('div.form-group');
+
+  if (filtering_mode == '0') {
```

Comment:
Nitpick: could you please either add some comments to the conditions so that it's more obvious what the numbers mean?